Automattic/woocommerce-payments
# Add Tap to Pay icon on the transactions list page for Tap to Pay transactions

Fixes #5377

#### Changes proposed in this Pull Request
Adding a Tap to Pay icon on the transactions list page for Tap to Pay transactions.


#### Testing instructions
- Switch to the branch 3080-gh-Automattic/woocommerce-payments-server
- Create an order and pay it with Tap to Pay
- Navigate to Transactions list page
- Verify a Tap to Pay icon is displayed next to the `In Person` value in `Channel` column. 


-------------------


**Post merge**
